bookshelf/6-gce/makeBookshelf

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 #
-#    Copyright 2016 Google, Inc.
+#    Copyright 2018, Google LLC.
```

Comment:
From my understanding the dates on headers aren't updated just because the file is.